app/models/video.rb

```diff
@@ -1,11 +1,24 @@
 class Video < ActiveRecord::Base
   belongs_to :category
+  has_many :reviews,-> { order "created_at DESC" }
 
   validates_presence_of :title, :description
 
   def self.search_by_title(search_term)
-    # % shows wildcard location
     return [] if search_term.blank?
     where('title ILIKE ?', "%#{search_term}%").order("created_at DESC")
   end
+
+  def average_rating
```

Comment:
You can refactor whole method with ActiveRecords#average method - check this  http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-average